src/anm/animation/element.js

```diff
@@ -1920,10 +1865,10 @@ Element.prototype.offset = function() {
 };
 
 /*Element.prototype.local = function(pt) {
-    this.matrix.transformPoint();
+    return this.matrix.transformPoint(pt);
 }
 Element.prototype.global = function(pt) {
-    this.matrix.transformPoint();
+    return this.matrix.adaptPoint(pt);
```

Comment:
It reminds me to add these methods finally, I just haven't finally invented the best/proper way to do it, but yeah.